### DISCUSSION TITLE
No webui for google colab.

### DISCUSSION THREAD

**Discussion by D0N-C4RV4LEZ** [nolanaatama/sd-1click-colab#39] [General]
Free colab version concerned not pro!
Watch these interesting discussions on the subject below!

https://www.reddit.com/r/StableDiffusion/comments/12t8tc7/is_colab_going_to_start_banning_people_who_use_it/
https://twitter.com/thechrisperry/status/1649189902079381505
https://twitter.com/artificialguybr/status/1649178893020979201

**Comment by ssolikhin** [comment]
For this last couple days I solved the warning problem by encoding the code into base64 and use 'eval' command.. I think it is because Googlecolab just detect some keywords that present in the code, so by encoding all the codes it there is no detection. I hope this trick lasts forever.
